cf/sns.yaml

```diff
@@ -33,26 +33,6 @@ Resources:
     Properties: 
       KmsMasterKeyId: "alias/aws/sns"
       TopicName: !Sub "work-results-${Environment}-${SandboxID}"
-  WorkResultsSubscription:
-    Type: 'AWS::SNS::Subscription'
-    Properties:
-      TopicArn: !Ref SNSTopic
-      Endpoint: !If [isProd, "https://api.scp.biomage.net/v1/workResults",
-                !If [isDev, "http://host.docker.internal:3000/v1/workResults",
-                !Sub "https://api-${SandboxID}.scp-${Environment}.biomage.net/v1/workResults"]]
```

Comment:
@kafkasl is this the subsription endpoint that you are making the state machine to call when it wants a pipeline pod assigned?

Reply:
I'm using the pipeline endpoint so it can be removed.

Reply:
Cool and I just double checked the worker- no sns is used anywhere (apart from in the tests and in the readme which we will need to clean up). So this code looks good.